cli/src/commands/config.rs

```diff
@@ -94,6 +100,9 @@ pub(crate) struct ConfigListArgs {
     /// Target the repo-level config
     #[arg(long)]
     repo: bool,
+    /// Target the workspace-level config
+    #[arg(long)]
+    workspace: bool,
```

Comment:
You'll need to update the `"specific"` list.

This will probably help:
https://github.com/martinvonz/jj/pull/3674

Reply:
Updated